lang/cs/Org.Apache.REEF.Driver/Bridge/Events/FailedTask.cs

```diff
@@ -87,16 +91,26 @@ public Optional<IActiveContext> GetActiveContext()
             return Optional<IActiveContext>.Of(new ActiveContext(ActiveContextClr2Java));
         }
 
+        /// <summary>
+        /// Returns the Exception causing the Failed Task.
+        /// If the Exception was caused by a control flow error (start, stop, suspend), 
```

Comment:
Please either use `<remarks>` or `<returns>` for these descriptions, as they are too long for a ToolTip :)